docusearch/models.py

```diff
@@ -28,7 +28,14 @@ class Document(models.Model):
     text = models.TextField(
         null=False, help_text='The full text of the document')
     title = models.TextField(null=True)
-    date = models.DateField(null=True)
+    date_created = models.DateField(
+        null=True, help_text='Date the document was created by agency')
+    date_released = models.DateField(
```

Comment:
Oh, we might not know the date the document was released, so I made this nullable. Eventually we'll know when we picked it up (and when we last looked at the website, data store) but making this nullable is going to give us options later.